piemap.yaml

```diff
@@ -0,0 +1,40 @@
+packages:
+  - sp
+  - maps
+  - maptools
+  - rgeos 
+  - stringr
+  - dataRetrieval
+  - lubridate
+  - dplyr
+  - mapdata
+  - jsonlite
+  - geojsonio
+
+sources:
+  - scripts/piemap/map_utils.R
+  - scripts/piemap/data_utils.R
+  - scripts/piemap/draw_utils.R
+  
+targets:
+  all: 
+    depends: 
+      - figures/national_pies.png
+  
+  plot_metadata:
+    command: c(I(7), I(4.3), I(500))
```

Comment:
could you add some argument names to this one so we can quickly see what the numbers are for?